cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AuthorizationModule.java

```diff
@@ -122,7 +122,7 @@ public DynamicDatasetCache get() {
       return new MultiThreadDatasetCache(
         dsInstantiator, txClient, NamespaceId.SYSTEM, ImmutableMap.<String, String>of(),
         metricsCollectionService.getContext(ImmutableMap.<String, String>of()),
-        ImmutableMap.<String, Map<String, String>>of()
+        ImmutableMap.<String, Map<String, String>>of(), "Authorization"
```

Comment:
I can change it for this one. Most of the other uses from CDAP are in system Datasets - DefaultConfigStore, MDS etc. Any suggested naming convention for them? 'cdap.system.dataset.DefaultConfigStore'?